src/sciwrappers.c

```diff
@@ -93,6 +93,10 @@ sptr_t sci_send_message_internal (const gchar *file, guint line, ScintillaObject
 }
 #endif
 
+ /* Set multi paste setting (at 3104 scintilla defaults to SC_MULTIPASTE_ONCE) */ 
+void sci_set_multipaste(ScintillaObject *sci, gint mpval) { 
+	SSM(sci, SCI_SETMULTIPASTE, mpval, 0); 
+}
```

Comment:
We don't actually need a new wrapper for just one call, especially not when that call is in *editor.c*.

Reply:
The reason I did this is because @elextr suggested that it should be an option. I agree that it should be the only behavior.

Reply:
Related discussion can be found in #2317 and in #850